docs/concepts/resource-limits-and-performance-optimization.md

```diff
@@ -36,22 +42,23 @@ If any content or task pane add-in exceeds the preceding thresholds on CPU core
 - Restart the add-in.
 - Cancel further alerts about exceeding that threshold. Ideally, the user should then delete the add-in from the document. Continued use of the add-in would risk further performance and stability issues.
 
-### Outlook add-ins
+### Evaluation response time for regular expressions in Outlook add-ins
 
-If any Outlook add-in exceeds the preceding thresholds for CPU core or memory usage, or tolerance limit for crashes, the add-in becomes unavailable. The Exchange Admin Center displays the add-in's status.
+Outlook add-ins that use regular expressions and run in Outlook on Windows (classic) or on Mac (classic) should observe the following rules on activation.
 
-> [!NOTE]
-> Even though only Outlook on Windows (classic) and on Mac monitor resource usage, if either of these clients makes an Outlook add-in unavailable, that add-in also becomes unavailable in Outlook on the web, on mobile devices, and in [new Outlook on Windows](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627).
+- **Regular expressions response time** - Applies to classic Outlook on Windows only. A default threshold of 1,000 milliseconds for Outlook to evaluate all regular expressions in the manifest of an Outlook add-in. Exceeding the threshold causes Outlook to retry evaluation at a later time.
 
-In addition to the CPU core, memory, and reliability rules, Outlook add-ins should observe the following rules on activation.
+  Administrators can adjust this default threshold value of 1,000 milliseconds by using a group policy or application-specific setting for the `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Outlook\ActivationAlertThreshold` DWORD value in the Windows registry.
 
-- **Regular expressions response time** - A default threshold of 1,000 milliseconds for Outlook to evaluate all regular expressions in the manifest of an Outlook add-in. Exceeding the threshold causes Outlook to retry evaluation at a later time.
+- **Regular expressions re-evaluation** - A default limit of three times for Outlook to reevaluate all the regular expressions in a manifest. If evaluation fails to evaluate within the time limit three times, Outlook makes the add-in unavailable.
```

Comment:
Hmm...maybe something like "evaluation fails to complete" so that we can avoid "repeating" the term "evaluate"?